e2e/pages/events/draftCaseManagementOrderEvent.page.js

```diff
@@ -1,4 +1,5 @@
 const { I } = inject();
+const draftDirections = require('../../fragments/draftDirections');
```

Comment:
What is `draftDirections`? Is that an object containing directions draft? I would not have doubts if constant was named right way.